docusaurus/docs/Android/02-client/04-events.md

```diff
@@ -1,5 +1,8 @@
 # Events
 
+Events allow the client to stay up to date with changes to the chat. Examples are a new message, a user's image that updated, a reaction, or a member joining the channel.
+Events are only received when SDK is connected to the socket and the application is in the foreground.
```

Comment:
```suggestion
Events are only received when the client is connected to the socket and the application is in the foreground.
```

Plus we could also link to push notifications here, as that's what you get if the socket connection is not active